manifests/base.pp

```diff
@@ -36,6 +36,13 @@
     }
   }
 
+  if $rsyslog::manage_service {
```

Comment:
There's no real need to move this.

Reply:
@dhollinger I had a go at fixing this early this morning (UK time).  My intention was to commit and push as an alternative PR when I got time over lunch (before you woke up!).  That never happened!  Sorry for not dropping a note on IRC to say I had something else in the works.

Anyway... in https://github.com/voxpupuli/puppet-rsyslog/pull/178 I do the same tidy up for `require_package`.  I've kept your tests unaltered.